code/apps/Managed Software Center/Managed Software Center/Controllers/MainWindowController.swift

```diff
@@ -1670,6 +1701,16 @@ extension MainWindowController: NSOutlineViewDelegate {
             }
             if let imageView = view?.imgView {
                 imageView.image = NSImage(named: NSImage.Name(icon))?.tint(color: .secondaryLabelColor)
+                if #available(macOS 11.0, *) {
```

Comment:
What does this mean if an admin has a custom sidebar item and the user is running < macOS 11?

Reply:
SF symbol only works with macOS 11.0 and above. Therefore the restriction. With older systems no symbol is displayed.
That could definitely be solved a little better

Reply:
Or just declare no support for custom sidebar items on macOS < 11.